### DISCUSSION TITLE
kudos!

### DISCUSSION THREAD

**Discussion by androbi-com** [NVIDIA/stdexec#731] [General]
Just wanted to drop a TYVM here! I have been searching for what modern C++ offers for concurrency/parallelism and was not really satisfied. Finally I found the P2300 proposal and this implementation and the expressiveness of pipelined senders invoking lambdas is really fantastic! Even distributed MPI communications and GPUs fit nicely in the picture. Thanks!

**Comment by npuichigo** [comment]
also coroutines😀

**Comment by ericniebler** [comment]
Thanks for the kudos! It is very much appreciated.
